src/services/dashboards/dashboards.svc.coffee

```diff
@@ -110,26 +110,47 @@ angular
             (success)->
               orgId = success.currentOrganization.id
 
+              # Retrieve dashboards
               $http.get(ImpacRoutes.dashboards.index(orgId)).then(
                 (dashboards)->
                   _self.setDashboards(dashboards.data).then(->
                     _self.setCurrentDashboard()
                     deferred.resolve(_self.config)
                     $log.info("Impac! - DashboardsSvc: loaded (force=#{force})")
 
-                  ).finally( -> _self.loadLocked=false )
+                  ).finally( -> _self.loadLocked = false )
```

Comment:
I think this finally may resolve before the retrieval of templates below is finished, unlocking the `loadLocked`.